applications/asset_tracker_v2/README.rst

```diff
@@ -69,7 +69,7 @@ The application supports the following data types:
 | Battery       | Voltage                    | APP_DATA_BATTERY                              |
 +---------------+----------------------------+-----------------------------------------------+
 
-The sets of sensor data that are published to the cloud service consist of relative timestamps that originate from the time of sampling.
+The sets of sensor data that are published to the cloud service consist of relative `timestamps <timestamping_>`_ that originate from the time of sampling.
```

Comment:
Should it be <Timestamping> as it is capitalized in the links.txt file?

Reply:
Updated.